modules/ROOT/pages/dataweave-formats.adoc

```diff
@@ -216,6 +206,19 @@ format-specific writer, the CSV writer, outputs the payload in CSV format:
 </file:write>
 ----
 
+Starting in Mule 4.3.0, you can set the `output` directive using the format ID alone,
+instead of using the MIME type. For example, you might set the `output` header directive
+of an expression in the Transform Message component or a Write operation to
+`output json` or `output csv`. You can also use the format ID to
+differentiate the format of the output data from the MIME type in the output header, and 
+you can use a custom MIME type. 
+For example, you can write JSON data but customize the MIME type to
+`application/problem+json` by using the DataWeave directive `output application/problem+json with json`.
+See xref:dataweave-cookbook-change-script-output-mime.adoc[Change a Script's MIME Type Output]
+for examples that customize the MIME type output of a script.
+ 
+The `with` keyword separates the output format from the DataWeave writer for the script's output. For example, you can specify 'output application/csv with binary' to output the `application/csv` format when using the `binary` writer. This setting is necessary in cases shown in xref:dataweave-cookbook-set-reader-writer-props.adoc#ex04[Use a Mule Variable as a Configuration Value] and xref:dataweave-cookbook-set-reader-writer-props.adoc#use-reader-and-writer-properties-in-dataweave-functions[Use Reader and Writer Properties in DataWeave Functions]. It is also possible to append writer properties, such as in the use case `output application/my-custom-type with json indent=false`. 
```

Comment:
```suggestion
The `with` keyword separates the output MIME type from the DataWeave writer for the script's output. For example, you can specify 'output application/csv with binary' to output the `application/csv` MIME type when using the `binary` writer. This setting is necessary in cases shown in xref:dataweave-cookbook-set-reader-writer-props.adoc#ex04[Use a Mule Variable as a Configuration Value] and xref:dataweave-cookbook-set-reader-writer-props.adoc#use-reader-and-writer-properties-in-dataweave-functions[Use Reader and Writer Properties in DataWeave Functions]. It is also possible to append writer properties, such as in the use case `output application/my-custom-type with json indent=false`. 
```
Just to maintain consistency with the sentence at line 212. I think the writer here is the format and 'aplication/blabla' the mime type.

Reply:
done